python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_hsic_estimators_ishigami.py

```diff
@@ -0,0 +1,262 @@
+"""
+The HSIC sensitivity indices: the Ishigami model
+================================================
+"""
+import openturns as ot
+import openturns.viewer as otv
+import numpy as np
+
+# %%
+#
+# This example is a brief overview of the HSIC classes and how to call them. HSIC estimators
+# are based on reproducing Hilbert-Schmidt kernel. We can use them to compute sensitivity
+# indices. We present the methods on the Ishigami model.
+#
+
+
+# %%
+# Definition of the model
+# -----------------------
+#
+# We first define the marginals and the 3d distribution of the input for the Ishigami model.
+X1 = ot.Uniform(-np.pi, np.pi)
+X2 = ot.Uniform(-np.pi, np.pi)
+X3 = ot.Uniform(-np.pi, np.pi)
+distX = ot.ComposedDistribution([X1, X2, X3])
+
+
+# %%
+# We generate an input sample of size 100 and dimension 3.
+size = 100
+X = distX.getSample(size)
+
+# %%
+# We define the Ishigami model and generate the output sample.
+modelIshigami = ot.SymbolicFunction(
+    ["X1", "X2", "X3"], ["sin(X1) + 5.0 * (sin(X2))^2 + 0.1 * X3^4 * sin(X1)"])
+Y = modelIshigami(X)
+
+
+# %%
+# Setting the covariance models
+# -----------------------------
+#
+# The HSIC algorithms use reproducing Hilbert-Schmidt kernels to estimate independence.
+# For each input variable we choose a covariance kernel. Here we choose a SquaredExponential
+# for all input variables.
```

Comment:
```suggestion
# For each input variable we choose a covariance kernel.
# Here we choose a :class:`~openturns.SquaredExponential`
# kernel for all input variables.
```
Links to the API doc are helpful, and you already created one to the `DistanceToDomainFunction` doc page, so creating others would be even better!